packages/fuselage/src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,100 @@
+import type { AriaAttributes } from 'react';
+import React, { useRef } from 'react';
+import { useNumberFormatter, useSlider } from 'react-aria';
+import { useSliderState } from 'react-stately';
+
+import { SliderHead } from './SliderHead';
+import { SliderThumb } from './SliderThumb';
+import { SliderTrack } from './SliderTrack';
+
+type SliderProps<T = number | number[]> = AriaAttributes & {
+  /**
+   * The display format of the value output.
+   */
+  formatOptions?: Intl.NumberFormatOptions;
+  label?: string;
+  showOutput?: boolean;
+  /**
+   * Slider with multiple thumbs.
+   * @default false
+   */
+  multiThumb?: T extends number[] ? true : false;
+  step?: number;
+  /**
+   * @default 0
+   */
+  minValue?: number;
+  /**
+   * @default 100
+   */
+  maxValue?: number;
+  orientation?: 'horizontal' | 'vertical';
+  isDisabled?: boolean;
```

Comment:
`is-` prefix is fine for method names, maybe for variable names also, but it becomes odd for object properties.

```suggestion
  disabled?: boolean;
```

Reply:
that's the prop name expected by `useSlider`... 
I could manipulate it before passing if you think it's necessary